FFXIVClientStructs/FFXIV/Component/GUI/AtkTextNode.cs

```diff
@@ -128,3 +126,33 @@ public enum FontType : byte {
     Jupiter = 0x4,
     JupiterLarge = 0x5,
 }
+
+[StructLayout(LayoutKind.Explicit, Size = 0x38)]
+public unsafe struct LinkData {
+    [FieldOffset(0x00)] public int Unk0;
+    [FieldOffset(0x04)] public int Unk4;
+    [FieldOffset(0x08)] public int Unk8;
+    [FieldOffset(0x0C)] public int UnkC;
+    /// <remarks> To be read until link terminator. </remarks>
+    [FieldOffset(0x10)] public byte* Payload;
+    /// <remarks> First 3 bytes are PayloadEnd(?) and last byte is <see cref="LinkType"/> </remarks>
+    [FieldOffset(0x18), CExportIgnore] public uint PayloadEndAndLinkType;
+    /// <remarks> The type of the Link payload. See LinkMacroPayloadType in Lumina. </remarks>
+    [FieldOffset(0x1B)] public byte LinkType;
+    [FieldOffset(0x1C)] public ushort Unk1C;
+    [FieldOffset(0x1E)] public ushort Unk1E;
+    [FieldOffset(0x20)] public uint Unk20;
+    // These are the 3 link payload parameters. Usually SeStrings have int expressions.
+    [FieldOffset(0x24)] public int IntValue1;
+    [FieldOffset(0x24), CExportIgnore] public uint UIntValue1;
+    [FieldOffset(0x28)] public int IntValue2;
+    [FieldOffset(0x28), CExportIgnore] public uint UIntValue2;
+    [FieldOffset(0x2C)] public int IntValue3;
+    [FieldOffset(0x2C), CExportIgnore] public uint UIntValue3;
```

Comment:
```suggestion
    [FieldOffset(0x24), CExporterUnion("Value1")] public int IntValue1;
    [FieldOffset(0x24), CExporterUnion("Value1")] public uint UIntValue1;
    [FieldOffset(0x28), CExporterUnion("Value2")] public int IntValue2;
    [FieldOffset(0x28), CExporterUnion("Value2")] public uint UIntValue2;
    [FieldOffset(0x2C), CExporterUnion("Value3")] public int IntValue3;
    [FieldOffset(0x2C), CExporterUnion("Value3")] public uint UIntValue3;
```

Use that if it depends which one it uses based on int or uint.

Another case could be that they are doing type casting in the code and there is only one value type